graph/repo.go

```diff
@@ -35,10 +35,10 @@ func TryMakeURI(cloneURL string) (string, error) {
 		return "", fmt.Errorf("MakeURI(%q): %s", cloneURL, err)
 	}
 
-	path := strings.TrimSuffix(url.Path, ".git")
-	path = filepath.Clean(path)
```

Comment:
Yep, this is a good change. URLs are always forward-slash separated paths, hence `path` package should be used, not `filepath`.